src/pages/paras/index.tsx

```diff
@@ -0,0 +1,418 @@
+import {
+  Backdrop,
+  Box,
+  Button,
+  CircularProgress,
+  FormControlLabel,
+  Paper,
+  Switch,
+  Table,
+  TableBody,
+  TableCell,
+  tableCellClasses,
+  TableContainer,
+  TableFooter,
+  TableHead,
+  TablePagination,
+  TableRow,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useRouter } from 'next/router';
+import React, { useEffect, useState } from 'react';
+
+import { useRenewableParachains } from '@/hooks/renewableParas';
+import { parseHNString } from '@/utils/functions';
+
+import { ActionButton, RegisterModal, ReserveModal } from '@/components';
+
+import chainData from '@/chaindata';
+import { useAccounts } from '@/contexts/account';
+import { useRelayApi } from '@/contexts/apis';
+import { ApiState } from '@/contexts/apis/types';
+import { useNetwork } from '@/contexts/network';
+
+// eslint-disable-next-line no-unused-vars
+enum ParaState {
+  // eslint-disable-next-line no-unused-vars
+  RESERVED = 'Reserved',
+  // eslint-disable-next-line no-unused-vars
+  ONBOARDING = 'Onboarding',
+  // eslint-disable-next-line no-unused-vars
+  PARATHREAD = 'Parathread',
+  // eslint-disable-next-line no-unused-vars
+  IDLE_PARA = 'Parachain(Idle)',
+  // eslint-disable-next-line no-unused-vars
+  ACTIVE_PARA = 'Parachain(Active)',
+}
+
+type ParachainInfo = {
+  id: number;
+  state: ParaState;
+  name: string;
+};
+
+const StateCard = ({ state }: { state: ParaState }) => {
+  const styles = {
+    [ParaState.RESERVED]: {
+      color: '#008000',
+      background: 'rgba(0, 128, 0, 0.1)',
+    },
+    [ParaState.ONBOARDING]: {
+      color: '#7472D8',
+      background: 'rgba(116, 114, 216, 0.1)',
+    },
+    [ParaState.PARATHREAD]: {
+      color: '#2D57C3',
+      background: 'rgba(45, 87, 195, 0.1)',
+    },
+    [ParaState.IDLE_PARA]: {
+      color: '#008000',
+      background: 'rgba(0, 128, 0, 0.1)',
+    },
+    [ParaState.ACTIVE_PARA]: {
+      color: '#9F53FF',
+      background: '#EDDFFF',
+    },
+  };
+  return (
+    <Typography
+      sx={{
+        color: styles[state].color,
+        bgcolor: styles[state].background,
+        padding: '0.5rem 1.25rem',
+        borderRadius: '1rem',
+        width: 'min-content',
+      }}
+    >
+      {state}
+    </Typography>
+  );
+};
+
+const ParachainManagement = () => {
+  const theme = useTheme();
+
+  const router = useRouter();
+
+  const {
+    state: { api: relayApi, apiState: relayApiState },
+  } = useRelayApi();
+  const { parachains: activeParas } = useRenewableParachains();
```

Comment:
Renewable paras have nothing to do with active paras. Active paras are the ones that can be found assigned within broker.workload.

If a parachain is not in broker.workload but can be found assigned in broker.workplan, it should be set to a new state called 'Soon Active'. In this state, no action should be requested.